--- pyart/map/_gate_to_grid_map.c.orig	2015-06-09 13:14:18.819307097 -0500
+++ pyart/map/_gate_to_grid_map.c	2015-06-09 13:15:14.747307509 -0500
@@ -4686,7 +4686,7 @@
  *                 # reflectivity filtering for non-finite and large values
  *                 if filter_flag:             # <<<<<<<<<<<<<<
  *                     value = field_data[nray, ngate, 0]
- *                     if not isfinite(value):
+ *                     if value > max_refl:
  */
       __pyx_t_18 = (__pyx_v_filter_flag != 0);
       if (__pyx_t_18) {
@@ -4695,7 +4695,7 @@
  *                 # reflectivity filtering for non-finite and large values
  *                 if filter_flag:
  *                     value = field_data[nray, ngate, 0]             # <<<<<<<<<<<<<<
- *                     if not isfinite(value):
+ *                     if value > max_refl:
  *                         continue
  */
         __pyx_t_20 = __pyx_v_nray;
@@ -4706,26 +4706,6 @@
         /* "pyart/map/_gate_to_grid_map.pyx":313
  *                 if filter_flag:
  *                     value = field_data[nray, ngate, 0]
- *                     if not isfinite(value):             # <<<<<<<<<<<<<<
- *                         continue
- *                     if value > max_refl:
- */
-        __pyx_t_18 = ((!(isfinite(__pyx_v_value) != 0)) != 0);
-        if (__pyx_t_18) {
-
-          /* "pyart/map/_gate_to_grid_map.pyx":314
- *                     value = field_data[nray, ngate, 0]
- *                     if not isfinite(value):
- *                         continue             # <<<<<<<<<<<<<<
- *                     if value > max_refl:
- *                         continue
- */
-          goto __pyx_L7_continue;
-        }
-
-        /* "pyart/map/_gate_to_grid_map.pyx":315
- *                     if not isfinite(value):
- *                         continue
  *                     if value > max_refl:             # <<<<<<<<<<<<<<
  *                         continue
  * 
@@ -4733,8 +4713,8 @@
         __pyx_t_18 = ((__pyx_v_value > __pyx_v_max_refl) != 0);
         if (__pyx_t_18) {
 
-          /* "pyart/map/_gate_to_grid_map.pyx":316
- *                         continue
+          /* "pyart/map/_gate_to_grid_map.pyx":314
+ *                     value = field_data[nray, ngate, 0]
  *                     if value > max_refl:
  *                         continue             # <<<<<<<<<<<<<<
  * 
@@ -4746,7 +4726,7 @@
       }
       __pyx_L11:;
 
-      /* "pyart/map/_gate_to_grid_map.pyx":318
+      /* "pyart/map/_gate_to_grid_map.pyx":316
  *                         continue
  * 
  *                 values = field_data[nray, ngate]             # <<<<<<<<<<<<<<
@@ -4765,7 +4745,7 @@
         __pyx_tmp_idx += __pyx_tmp_shape;
     if (0 && (__pyx_tmp_idx < 0 || __pyx_tmp_idx >= __pyx_tmp_shape)) {
         PyErr_SetString(PyExc_IndexError, "Index out of bounds (axis 0)");
-        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 318; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
+        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 316; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
     }
         __pyx_t_23.data += __pyx_tmp_idx * __pyx_tmp_stride;
 }
@@ -4778,7 +4758,7 @@
         __pyx_tmp_idx += __pyx_tmp_shape;
     if (0 && (__pyx_tmp_idx < 0 || __pyx_tmp_idx >= __pyx_tmp_shape)) {
         PyErr_SetString(PyExc_IndexError, "Index out of bounds (axis 1)");
-        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 318; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
+        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 316; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
     }
         __pyx_t_23.data += __pyx_tmp_idx * __pyx_tmp_stride;
 }
@@ -4792,7 +4772,7 @@
       __pyx_t_23.memview = NULL;
       __pyx_t_23.data = NULL;
 
-      /* "pyart/map/_gate_to_grid_map.pyx":319
+      /* "pyart/map/_gate_to_grid_map.pyx":317
  * 
  *                 values = field_data[nray, ngate]
  *                 masks = field_mask[nray, ngate]             # <<<<<<<<<<<<<<
@@ -4811,7 +4791,7 @@
         __pyx_tmp_idx += __pyx_tmp_shape;
     if (0 && (__pyx_tmp_idx < 0 || __pyx_tmp_idx >= __pyx_tmp_shape)) {
         PyErr_SetString(PyExc_IndexError, "Index out of bounds (axis 0)");
-        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 319; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
+        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 317; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
     }
         __pyx_t_25.data += __pyx_tmp_idx * __pyx_tmp_stride;
 }
@@ -4824,7 +4804,7 @@
         __pyx_tmp_idx += __pyx_tmp_shape;
     if (0 && (__pyx_tmp_idx < 0 || __pyx_tmp_idx >= __pyx_tmp_shape)) {
         PyErr_SetString(PyExc_IndexError, "Index out of bounds (axis 1)");
-        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 319; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
+        {__pyx_filename = __pyx_f[0]; __pyx_lineno = 317; __pyx_clineno = __LINE__; goto __pyx_L1_error;}
     }
         __pyx_t_25.data += __pyx_tmp_idx * __pyx_tmp_stride;
 }
@@ -4838,7 +4818,7 @@
       __pyx_t_25.memview = NULL;
       __pyx_t_25.data = NULL;
 
-      /* "pyart/map/_gate_to_grid_map.pyx":320
+      /* "pyart/map/_gate_to_grid_map.pyx":318
  *                 values = field_data[nray, ngate]
  *                 masks = field_mask[nray, ngate]
  *                 self.map_gate(x, y, z, roi, values, masks, weighting_function)             # <<<<<<<<<<<<<<
@@ -4883,7 +4863,7 @@
   return __pyx_r;
 }
 
-/* "pyart/map/_gate_to_grid_map.pyx":326
+/* "pyart/map/_gate_to_grid_map.pyx":324
  *     @cython.boundscheck(False)
  *     @cython.wraparound(False)
  *     cdef int map_gate(self, float x, float y, float z, float roi,             # <<<<<<<<<<<<<<
@@ -4931,7 +4911,7 @@
   int __pyx_t_19;
   __Pyx_RefNannySetupContext("map_gate", 0);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":336
+  /* "pyart/map/_gate_to_grid_map.pyx":334
  * 
  *         # shift positions so that grid starts at 0
  *         x -= self.x_start             # <<<<<<<<<<<<<<
@@ -4940,7 +4920,7 @@
  */
   __pyx_v_x = (__pyx_v_x - __pyx_v_self->x_start);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":337
+  /* "pyart/map/_gate_to_grid_map.pyx":335
  *         # shift positions so that grid starts at 0
  *         x -= self.x_start
  *         y -= self.y_start             # <<<<<<<<<<<<<<
@@ -4949,7 +4929,7 @@
  */
   __pyx_v_y = (__pyx_v_y - __pyx_v_self->y_start);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":338
+  /* "pyart/map/_gate_to_grid_map.pyx":336
  *         x -= self.x_start
  *         y -= self.y_start
  *         z -= self.z_start             # <<<<<<<<<<<<<<
@@ -4958,7 +4938,7 @@
  */
   __pyx_v_z = (__pyx_v_z - __pyx_v_self->z_start);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":340
+  /* "pyart/map/_gate_to_grid_map.pyx":338
  *         z -= self.z_start
  * 
  *         x_min = find_min(x, roi, self.x_step)             # <<<<<<<<<<<<<<
@@ -4967,7 +4947,7 @@
  */
   __pyx_v_x_min = __pyx_f_5pyart_3map_17_gate_to_grid_map_find_min(__pyx_v_x, __pyx_v_roi, __pyx_v_self->x_step);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":341
+  /* "pyart/map/_gate_to_grid_map.pyx":339
  * 
  *         x_min = find_min(x, roi, self.x_step)
  *         if x_min > self.nx-1:             # <<<<<<<<<<<<<<
@@ -4977,7 +4957,7 @@
   __pyx_t_1 = ((__pyx_v_x_min > (__pyx_v_self->nx - 1)) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":342
+    /* "pyart/map/_gate_to_grid_map.pyx":340
  *         x_min = find_min(x, roi, self.x_step)
  *         if x_min > self.nx-1:
  *             return 0             # <<<<<<<<<<<<<<
@@ -4988,7 +4968,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":343
+  /* "pyart/map/_gate_to_grid_map.pyx":341
  *         if x_min > self.nx-1:
  *             return 0
  *         x_max = find_max(x, roi, self.x_step, self.nx)             # <<<<<<<<<<<<<<
@@ -4997,7 +4977,7 @@
  */
   __pyx_v_x_max = __pyx_f_5pyart_3map_17_gate_to_grid_map_find_max(__pyx_v_x, __pyx_v_roi, __pyx_v_self->x_step, __pyx_v_self->nx);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":344
+  /* "pyart/map/_gate_to_grid_map.pyx":342
  *             return 0
  *         x_max = find_max(x, roi, self.x_step, self.nx)
  *         if x_max < 0:             # <<<<<<<<<<<<<<
@@ -5007,7 +4987,7 @@
   __pyx_t_1 = ((__pyx_v_x_max < 0) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":345
+    /* "pyart/map/_gate_to_grid_map.pyx":343
  *         x_max = find_max(x, roi, self.x_step, self.nx)
  *         if x_max < 0:
  *             return 0             # <<<<<<<<<<<<<<
@@ -5018,7 +4998,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":347
+  /* "pyart/map/_gate_to_grid_map.pyx":345
  *             return 0
  * 
  *         y_min = find_min(y, roi, self.y_step)             # <<<<<<<<<<<<<<
@@ -5027,7 +5007,7 @@
  */
   __pyx_v_y_min = __pyx_f_5pyart_3map_17_gate_to_grid_map_find_min(__pyx_v_y, __pyx_v_roi, __pyx_v_self->y_step);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":348
+  /* "pyart/map/_gate_to_grid_map.pyx":346
  * 
  *         y_min = find_min(y, roi, self.y_step)
  *         if y_min > self.ny-1:             # <<<<<<<<<<<<<<
@@ -5037,7 +5017,7 @@
   __pyx_t_1 = ((__pyx_v_y_min > (__pyx_v_self->ny - 1)) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":349
+    /* "pyart/map/_gate_to_grid_map.pyx":347
  *         y_min = find_min(y, roi, self.y_step)
  *         if y_min > self.ny-1:
  *             return 0             # <<<<<<<<<<<<<<
@@ -5048,7 +5028,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":350
+  /* "pyart/map/_gate_to_grid_map.pyx":348
  *         if y_min > self.ny-1:
  *             return 0
  *         y_max = find_max(y, roi, self.y_step, self.ny)             # <<<<<<<<<<<<<<
@@ -5057,7 +5037,7 @@
  */
   __pyx_v_y_max = __pyx_f_5pyart_3map_17_gate_to_grid_map_find_max(__pyx_v_y, __pyx_v_roi, __pyx_v_self->y_step, __pyx_v_self->ny);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":351
+  /* "pyart/map/_gate_to_grid_map.pyx":349
  *             return 0
  *         y_max = find_max(y, roi, self.y_step, self.ny)
  *         if y_max < 0:             # <<<<<<<<<<<<<<
@@ -5067,7 +5047,7 @@
   __pyx_t_1 = ((__pyx_v_y_max < 0) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":352
+    /* "pyart/map/_gate_to_grid_map.pyx":350
  *         y_max = find_max(y, roi, self.y_step, self.ny)
  *         if y_max < 0:
  *             return 0             # <<<<<<<<<<<<<<
@@ -5078,7 +5058,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":354
+  /* "pyart/map/_gate_to_grid_map.pyx":352
  *             return 0
  * 
  *         z_min = find_min(z, roi, self.z_step)             # <<<<<<<<<<<<<<
@@ -5087,7 +5067,7 @@
  */
   __pyx_v_z_min = __pyx_f_5pyart_3map_17_gate_to_grid_map_find_min(__pyx_v_z, __pyx_v_roi, __pyx_v_self->z_step);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":355
+  /* "pyart/map/_gate_to_grid_map.pyx":353
  * 
  *         z_min = find_min(z, roi, self.z_step)
  *         if z_min > self.nz-1:             # <<<<<<<<<<<<<<
@@ -5097,7 +5077,7 @@
   __pyx_t_1 = ((__pyx_v_z_min > (__pyx_v_self->nz - 1)) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":356
+    /* "pyart/map/_gate_to_grid_map.pyx":354
  *         z_min = find_min(z, roi, self.z_step)
  *         if z_min > self.nz-1:
  *             return 0             # <<<<<<<<<<<<<<
@@ -5108,7 +5088,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":357
+  /* "pyart/map/_gate_to_grid_map.pyx":355
  *         if z_min > self.nz-1:
  *             return 0
  *         z_max = find_max(z, roi, self.z_step, self.nz)             # <<<<<<<<<<<<<<
@@ -5117,7 +5097,7 @@
  */
   __pyx_v_z_max = __pyx_f_5pyart_3map_17_gate_to_grid_map_find_max(__pyx_v_z, __pyx_v_roi, __pyx_v_self->z_step, __pyx_v_self->nz);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":358
+  /* "pyart/map/_gate_to_grid_map.pyx":356
  *             return 0
  *         z_max = find_max(z, roi, self.z_step, self.nz)
  *         if z_max < 0:             # <<<<<<<<<<<<<<
@@ -5127,7 +5107,7 @@
   __pyx_t_1 = ((__pyx_v_z_max < 0) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":359
+    /* "pyart/map/_gate_to_grid_map.pyx":357
  *         z_max = find_max(z, roi, self.z_step, self.nz)
  *         if z_max < 0:
  *             return 0             # <<<<<<<<<<<<<<
@@ -5138,7 +5118,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":361
+  /* "pyart/map/_gate_to_grid_map.pyx":359
  *             return 0
  * 
  *         roi2 = roi * roi             # <<<<<<<<<<<<<<
@@ -5147,7 +5127,7 @@
  */
   __pyx_v_roi2 = (__pyx_v_roi * __pyx_v_roi);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":362
+  /* "pyart/map/_gate_to_grid_map.pyx":360
  * 
  *         roi2 = roi * roi
  *         for xi in range(x_min, x_max+1):             # <<<<<<<<<<<<<<
@@ -5158,7 +5138,7 @@
   for (__pyx_t_3 = __pyx_v_x_min; __pyx_t_3 < __pyx_t_2; __pyx_t_3+=1) {
     __pyx_v_xi = __pyx_t_3;
 
-    /* "pyart/map/_gate_to_grid_map.pyx":363
+    /* "pyart/map/_gate_to_grid_map.pyx":361
  *         roi2 = roi * roi
  *         for xi in range(x_min, x_max+1):
  *             for yi in range(y_min, y_max+1):             # <<<<<<<<<<<<<<
@@ -5169,7 +5149,7 @@
     for (__pyx_t_5 = __pyx_v_y_min; __pyx_t_5 < __pyx_t_4; __pyx_t_5+=1) {
       __pyx_v_yi = __pyx_t_5;
 
-      /* "pyart/map/_gate_to_grid_map.pyx":364
+      /* "pyart/map/_gate_to_grid_map.pyx":362
  *         for xi in range(x_min, x_max+1):
  *             for yi in range(y_min, y_max+1):
  *                 for zi in range(z_min, z_max+1):             # <<<<<<<<<<<<<<
@@ -5180,7 +5160,7 @@
       for (__pyx_t_7 = __pyx_v_z_min; __pyx_t_7 < __pyx_t_6; __pyx_t_7+=1) {
         __pyx_v_zi = __pyx_t_7;
 
-        /* "pyart/map/_gate_to_grid_map.pyx":365
+        /* "pyart/map/_gate_to_grid_map.pyx":363
  *             for yi in range(y_min, y_max+1):
  *                 for zi in range(z_min, z_max+1):
  *                     xg = self.x_step * xi             # <<<<<<<<<<<<<<
@@ -5189,7 +5169,7 @@
  */
         __pyx_v_xg = (__pyx_v_self->x_step * __pyx_v_xi);
 
-        /* "pyart/map/_gate_to_grid_map.pyx":366
+        /* "pyart/map/_gate_to_grid_map.pyx":364
  *                 for zi in range(z_min, z_max+1):
  *                     xg = self.x_step * xi
  *                     yg = self.y_step * yi             # <<<<<<<<<<<<<<
@@ -5198,7 +5178,7 @@
  */
         __pyx_v_yg = (__pyx_v_self->y_step * __pyx_v_yi);
 
-        /* "pyart/map/_gate_to_grid_map.pyx":367
+        /* "pyart/map/_gate_to_grid_map.pyx":365
  *                     xg = self.x_step * xi
  *                     yg = self.y_step * yi
  *                     zg = self.z_step * zi             # <<<<<<<<<<<<<<
@@ -5207,7 +5187,7 @@
  */
         __pyx_v_zg = (__pyx_v_self->z_step * __pyx_v_zi);
 
-        /* "pyart/map/_gate_to_grid_map.pyx":368
+        /* "pyart/map/_gate_to_grid_map.pyx":366
  *                     yg = self.y_step * yi
  *                     zg = self.z_step * zi
  *                     dist2 = (xg-x)*(xg-x) + (yg-y)*(yg-y) + (zg-z)*(zg-z)             # <<<<<<<<<<<<<<
@@ -5216,7 +5196,7 @@
  */
         __pyx_v_dist2 = ((((__pyx_v_xg - __pyx_v_x) * (__pyx_v_xg - __pyx_v_x)) + ((__pyx_v_yg - __pyx_v_y) * (__pyx_v_yg - __pyx_v_y))) + ((__pyx_v_zg - __pyx_v_z) * (__pyx_v_zg - __pyx_v_z)));
 
-        /* "pyart/map/_gate_to_grid_map.pyx":370
+        /* "pyart/map/_gate_to_grid_map.pyx":368
  *                     dist2 = (xg-x)*(xg-x) + (yg-y)*(yg-y) + (zg-z)*(zg-z)
  * 
  *                     if dist2 > roi2:             # <<<<<<<<<<<<<<
@@ -5226,7 +5206,7 @@
         __pyx_t_1 = ((__pyx_v_dist2 > __pyx_v_roi2) != 0);
         if (__pyx_t_1) {
 
-          /* "pyart/map/_gate_to_grid_map.pyx":371
+          /* "pyart/map/_gate_to_grid_map.pyx":369
  * 
  *                     if dist2 > roi2:
  *                         continue             # <<<<<<<<<<<<<<
@@ -5236,7 +5216,7 @@
           goto __pyx_L13_continue;
         }
 
-        /* "pyart/map/_gate_to_grid_map.pyx":373
+        /* "pyart/map/_gate_to_grid_map.pyx":371
  *                         continue
  * 
  *                     if weighting_function == BARNES:             # <<<<<<<<<<<<<<
@@ -5246,7 +5226,7 @@
         __pyx_t_1 = ((__pyx_v_weighting_function == __pyx_v_5pyart_3map_17_gate_to_grid_map_BARNES) != 0);
         if (__pyx_t_1) {
 
-          /* "pyart/map/_gate_to_grid_map.pyx":374
+          /* "pyart/map/_gate_to_grid_map.pyx":372
  * 
  *                     if weighting_function == BARNES:
  *                         weight = exp(-(dist2) / (2*roi2)) + 1e-5             # <<<<<<<<<<<<<<
@@ -5258,7 +5238,7 @@
         }
         /*else*/ {
 
-          /* "pyart/map/_gate_to_grid_map.pyx":376
+          /* "pyart/map/_gate_to_grid_map.pyx":374
  *                         weight = exp(-(dist2) / (2*roi2)) + 1e-5
  *                     else:   # CRESSMAN
  *                         weight = (roi2 - dist2) / (roi2 + dist2)             # <<<<<<<<<<<<<<
@@ -5269,7 +5249,7 @@
         }
         __pyx_L16:;
 
-        /* "pyart/map/_gate_to_grid_map.pyx":377
+        /* "pyart/map/_gate_to_grid_map.pyx":375
  *                     else:   # CRESSMAN
  *                         weight = (roi2 - dist2) / (roi2 + dist2)
  *                     for i in range(self.nfields):             # <<<<<<<<<<<<<<
@@ -5280,7 +5260,7 @@
         for (__pyx_t_9 = 0; __pyx_t_9 < __pyx_t_8; __pyx_t_9+=1) {
           __pyx_v_i = __pyx_t_9;
 
-          /* "pyart/map/_gate_to_grid_map.pyx":378
+          /* "pyart/map/_gate_to_grid_map.pyx":376
  *                         weight = (roi2 - dist2) / (roi2 + dist2)
  *                     for i in range(self.nfields):
  *                         if masks[i]:             # <<<<<<<<<<<<<<
@@ -5291,7 +5271,7 @@
           __pyx_t_1 = ((*((char *) ( /* dim=0 */ (__pyx_v_masks.data + __pyx_t_10 * __pyx_v_masks.strides[0]) ))) != 0);
           if (__pyx_t_1) {
 
-            /* "pyart/map/_gate_to_grid_map.pyx":379
+            /* "pyart/map/_gate_to_grid_map.pyx":377
  *                     for i in range(self.nfields):
  *                         if masks[i]:
  *                             continue             # <<<<<<<<<<<<<<
@@ -5301,7 +5281,7 @@
             goto __pyx_L17_continue;
           }
 
-          /* "pyart/map/_gate_to_grid_map.pyx":380
+          /* "pyart/map/_gate_to_grid_map.pyx":378
  *                         if masks[i]:
  *                             continue
  *                         self.grid_sum[zi, yi, xi, i] += weight * values[i]             # <<<<<<<<<<<<<<
@@ -5315,7 +5295,7 @@
           __pyx_t_15 = __pyx_v_i;
           *((float *) ( /* dim=3 */ ((char *) (((float *) ( /* dim=2 */ (( /* dim=1 */ (( /* dim=0 */ (__pyx_v_self->grid_sum.data + __pyx_t_12 * __pyx_v_self->grid_sum.strides[0]) ) + __pyx_t_13 * __pyx_v_self->grid_sum.strides[1]) ) + __pyx_t_14 * __pyx_v_self->grid_sum.strides[2]) )) + __pyx_t_15)) )) += (__pyx_v_weight * (*((float *) ( /* dim=0 */ (__pyx_v_values.data + __pyx_t_11 * __pyx_v_values.strides[0]) ))));
 
-          /* "pyart/map/_gate_to_grid_map.pyx":381
+          /* "pyart/map/_gate_to_grid_map.pyx":379
  *                             continue
  *                         self.grid_sum[zi, yi, xi, i] += weight * values[i]
  *                         self.grid_wsum[zi, yi, xi, i] += weight             # <<<<<<<<<<<<<<
@@ -5334,7 +5314,7 @@
     }
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":382
+  /* "pyart/map/_gate_to_grid_map.pyx":380
  *                         self.grid_sum[zi, yi, xi, i] += weight * values[i]
  *                         self.grid_wsum[zi, yi, xi, i] += weight
  *         return 1             # <<<<<<<<<<<<<<
@@ -5344,7 +5324,7 @@
   __pyx_r = 1;
   goto __pyx_L0;
 
-  /* "pyart/map/_gate_to_grid_map.pyx":326
+  /* "pyart/map/_gate_to_grid_map.pyx":324
  *     @cython.boundscheck(False)
  *     @cython.wraparound(False)
  *     cdef int map_gate(self, float x, float y, float z, float roi,             # <<<<<<<<<<<<<<
@@ -5358,7 +5338,7 @@
   return __pyx_r;
 }
 
-/* "pyart/map/_gate_to_grid_map.pyx":386
+/* "pyart/map/_gate_to_grid_map.pyx":384
  * 
  * @cython.cdivision(True)
  * cdef int find_min(float a, float roi, float step):             # <<<<<<<<<<<<<<
@@ -5373,7 +5353,7 @@
   int __pyx_t_1;
   __Pyx_RefNannySetupContext("find_min", 0);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":389
+  /* "pyart/map/_gate_to_grid_map.pyx":387
  *     """ Find the mimumum gate index for a dimension. """
  *     cdef int a_min
  *     if step == 0:             # <<<<<<<<<<<<<<
@@ -5383,7 +5363,7 @@
   __pyx_t_1 = ((__pyx_v_step == 0.0) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":390
+    /* "pyart/map/_gate_to_grid_map.pyx":388
  *     cdef int a_min
  *     if step == 0:
  *         return 0             # <<<<<<<<<<<<<<
@@ -5394,7 +5374,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":391
+  /* "pyart/map/_gate_to_grid_map.pyx":389
  *     if step == 0:
  *         return 0
  *     a_min = <int>ceil((a - roi) / step)             # <<<<<<<<<<<<<<
@@ -5403,7 +5383,7 @@
  */
   __pyx_v_a_min = ((int)ceil(((__pyx_v_a - __pyx_v_roi) / __pyx_v_step)));
 
-  /* "pyart/map/_gate_to_grid_map.pyx":392
+  /* "pyart/map/_gate_to_grid_map.pyx":390
  *         return 0
  *     a_min = <int>ceil((a - roi) / step)
  *     if a_min < 0:             # <<<<<<<<<<<<<<
@@ -5413,7 +5393,7 @@
   __pyx_t_1 = ((__pyx_v_a_min < 0) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":393
+    /* "pyart/map/_gate_to_grid_map.pyx":391
  *     a_min = <int>ceil((a - roi) / step)
  *     if a_min < 0:
  *         a_min = 0             # <<<<<<<<<<<<<<
@@ -5425,7 +5405,7 @@
   }
   __pyx_L4:;
 
-  /* "pyart/map/_gate_to_grid_map.pyx":394
+  /* "pyart/map/_gate_to_grid_map.pyx":392
  *     if a_min < 0:
  *         a_min = 0
  *     return a_min             # <<<<<<<<<<<<<<
@@ -5435,7 +5415,7 @@
   __pyx_r = __pyx_v_a_min;
   goto __pyx_L0;
 
-  /* "pyart/map/_gate_to_grid_map.pyx":386
+  /* "pyart/map/_gate_to_grid_map.pyx":384
  * 
  * @cython.cdivision(True)
  * cdef int find_min(float a, float roi, float step):             # <<<<<<<<<<<<<<
@@ -5449,7 +5429,7 @@
   return __pyx_r;
 }
 
-/* "pyart/map/_gate_to_grid_map.pyx":398
+/* "pyart/map/_gate_to_grid_map.pyx":396
  * 
  * @cython.cdivision(True)
  * cdef int find_max(float a, float roi, float step, int na):             # <<<<<<<<<<<<<<
@@ -5464,7 +5444,7 @@
   int __pyx_t_1;
   __Pyx_RefNannySetupContext("find_max", 0);
 
-  /* "pyart/map/_gate_to_grid_map.pyx":401
+  /* "pyart/map/_gate_to_grid_map.pyx":399
  *     """ Find the maximum gate index for a dimension. """
  *     cdef int a_max
  *     if step == 0:             # <<<<<<<<<<<<<<
@@ -5474,7 +5454,7 @@
   __pyx_t_1 = ((__pyx_v_step == 0.0) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":402
+    /* "pyart/map/_gate_to_grid_map.pyx":400
  *     cdef int a_max
  *     if step == 0:
  *         return 0             # <<<<<<<<<<<<<<
@@ -5485,7 +5465,7 @@
     goto __pyx_L0;
   }
 
-  /* "pyart/map/_gate_to_grid_map.pyx":403
+  /* "pyart/map/_gate_to_grid_map.pyx":401
  *     if step == 0:
  *         return 0
  *     a_max = <int>floor((a + roi) / step)             # <<<<<<<<<<<<<<
@@ -5494,7 +5474,7 @@
  */
   __pyx_v_a_max = ((int)floor(((__pyx_v_a + __pyx_v_roi) / __pyx_v_step)));
 
-  /* "pyart/map/_gate_to_grid_map.pyx":404
+  /* "pyart/map/_gate_to_grid_map.pyx":402
  *         return 0
  *     a_max = <int>floor((a + roi) / step)
  *     if a_max > na-1:             # <<<<<<<<<<<<<<
@@ -5504,7 +5484,7 @@
   __pyx_t_1 = ((__pyx_v_a_max > (__pyx_v_na - 1)) != 0);
   if (__pyx_t_1) {
 
-    /* "pyart/map/_gate_to_grid_map.pyx":405
+    /* "pyart/map/_gate_to_grid_map.pyx":403
  *     a_max = <int>floor((a + roi) / step)
  *     if a_max > na-1:
  *         a_max = na-1             # <<<<<<<<<<<<<<
@@ -5515,7 +5495,7 @@
   }
   __pyx_L4:;
 
-  /* "pyart/map/_gate_to_grid_map.pyx":406
+  /* "pyart/map/_gate_to_grid_map.pyx":404
  *     if a_max > na-1:
  *         a_max = na-1
  *     return a_max             # <<<<<<<<<<<<<<
@@ -5523,7 +5503,7 @@
   __pyx_r = __pyx_v_a_max;
   goto __pyx_L0;
 
-  /* "pyart/map/_gate_to_grid_map.pyx":398
+  /* "pyart/map/_gate_to_grid_map.pyx":396
  * 
  * @cython.cdivision(True)
  * cdef int find_max(float a, float roi, float step, int na):             # <<<<<<<<<<<<<<
